eng/dependabot/Packages.props

```diff
@@ -9,9 +9,9 @@
     <PackageReference Update="Microsoft.Build.Framework" Version="17.8.3" />
     <PackageReference Update="Microsoft.Build.Utilities.Core" Version="17.8.3" />
     <PackageReference Update="Newtonsoft.Json" Version="13.0.3" />
-    <PackageReference Update="Microsoft.Extensions.Logging" Version="7.0.0" />
-    <PackageReference Update="Microsoft.Extensions.Logging.Console" Version="7.0.0" />
-    <PackageReference Update="Microsoft.Extensions.Logging.Abstractions" Version="7.0.1" />
+    <PackageReference Update="Microsoft.Extensions.Logging" Version="8.0.0" />
```

Comment:
These new versions will need to be added to source-build-reference-packages and flow into this repo before the source-build leg will pass.  You can find the instructions for adding these new package versions [here](https://github.com/dotnet/source-build-reference-packages#adding-new-packages).